modules/standard/Path.chpl

```diff
@@ -67,7 +67,8 @@ module Path {
 
 use List;
 use SysError, IO;
-use Sys, OS.POSIX;
+use OS;
```

Comment:
I think `sys_getenv` is what's causing your use of `OS` here, so if you end up moving `getenv` to `OS.POSIX`, this may not be needed (and this module may want its own extern definition of `sys_getenv` so that it can continue to do the checks it was performing)